pangeo-deploy/values.yaml

```diff
@@ -7,34 +7,34 @@
 rbac:
   enabled: true
 
-
 # Dependency configuration values
 # -------------------------------
 # To configure dependencies you must create a key with the name of the dependency
 # and then configure as per the dependent chart's values.yaml. Don't forget to indent!
+pangeo:
+  jupyterhub:
+    # Helm config for jupyterhub goes here
+    # See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/jupyterhub/values.yaml
 
-jupyterhub:
-  # Helm config for jupyterhub goes here
-  # See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/jupyterhub/values.yaml
-
-  singleuser:
-    image:
-      name: pangeo/base-notebook
-      tag: 2019.02.01
-    cpu:
-      limit: 2
-      guarantee: 1
-    memory:
-      limit: 4G
-      guarantee: 2G
-    defaultUrl: "/lab"
-    cmd: jupyter-labhub
-    serviceAccountName: daskkubernetes
+    singleuser:
+      image:
+        name: pangeo/base-notebook
+        tag: 2019.02.01
+      cpu:
+        limit: 2
+        guarantee: 1
+      memory:
+        limit: 4G
+        guarantee: 2G
+      defaultUrl: "/lab"
+      cmd: jupyter-labhub
+      serviceAccountName: daskkubernetes
```

Comment:
I think we can remove basically all of this.

Reply:
Because it's present in the pangeo chart?